### ISSUE TITLE
Swift PM fails to find your windowsdk repository

### ISSUE THREAD

**Issue by paulober** [thebrowsercompany/windows-samples#21]
I tried to build your `thebrowsercompany/windows-samples` but I get following error:
```console
Updating https://github.com/thebrowsercompany/swift-windowsappsdk.git
Updated https://github.com/thebrowsercompany/swift-windowsappsdk.git (0.64s)
error: could not find a branch named ‘main’ in https://github.com/thebrowsercompany/swift-windowsappsdk.git
```

Also I used this swift build and since windowsappsdk package has issues disabled I opened it here.

**Comment by tristanlabelle** [comment]
Hmm I can't explain that since the repository definitely has a main branch: https://github.com/thebrowsercompany/swift-windowsappsdk/tree/main . I've just tried on a fresh clone and it works:

```
Fetching https://github.com/thebrowsercompany/swift-windowsappsdk
Fetched https://github.com/thebrowsercompany/swift-windowsappsdk from cache (1.04s)
Fetching https://github.com/thebrowsercompany/swift-windowsfoundation
Fetched https://github.com/thebrowsercompany/swift-windowsfoundation from cache (0.87s)
Fetching https://github.com/thebrowsercompany/swift-winui
Fetched https://github.com/thebrowsercompany/swift-winui from cache (1.03s)
Fetching https://github.com/thebrowsercompany/swift-cwinrt
Fetched https://github.com/thebrowsercompany/swift-cwinrt from cache (1.26s)
Fetching https://github.com/thebrowsercompany/swift-uwp
Fetched https://github.com/thebrowsercompany/swift-uwp from cache (0.92s)
Creating working copy for https://github.com/thebrowsercompany/swift-cwinrt
Working copy of https://github.com/thebrowsercompany/swift-cwinrt resolved at main (ad451d3)
Creating working copy for https://github.com/thebrowsercompany/swift-windowsappsdk
Working copy of https://github.com/thebrowsercompany/swift-windowsappsdk resolved at main (9706ffd)
Creating working copy for https://github.com/thebrowsercompany/swift-winui
Working copy of https://github.com/thebrowsercompany/swift-winui resolved at main (41e7b0f)
Creating working copy for https://github.com/thebrowsercompany/swift-uwp
Working copy of https://github.com/thebrowsercompany/swift-uwp resolved at main (cfad0a4)
Creating working copy for https://github.com/thebrowsercompany/swift-windowsfoundation
Working copy of https://github.com/thebrowsercompany/swift-windowsfoundation resolved at main (ba7a8b5)
Building for debugging...
```

Can you delete your `.build` folder and try again?

**Comment by tristanlabelle** [comment]
We've decided to archive this repository because it is outdated, and keep it for illustration purposes. Most Swift WinUI projects beyond the proof of concept phase will need to use https://github.com/thebrowsercompany/swift-winrt directly as to control which version of dependencies like WinUI, and which subset of APIs to generate projections for. We hope to provide better community support by consolidating on that repo.
